### PR TITLE
Dev enkogu sept

### PR DESCRIPTION
**Pull request by enkogu**

### CODE REVIEW

File: contracts/moneyflow/ether/WeiInfiniteFund.sol [hunk 1]
```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.4.24;
+import "./WeiFund.sol";
+
+
+contract WeiInfiniteFund is WeiFund {
```
Comment (AnthonyAkentiev) [inline contracts/moneyflow/ether/WeiInfiniteFund.sol:11]: Add comments for each new contract (in the std. format)

Comment (AnthonyAkentiev) [reply contracts/moneyflow/ether/WeiInfiniteFund.sol:11]: What is the idea behind the WeiInfiniteFund?
Why don't we have a WeiInfiniteExpense for example?

**Review by AnthonyAkentiev — CHANGES_REQUESTED** [review]

File: contracts/moneyflow/ether/WeiInfiniteFund.sol (hunk 1 above)
Comment (enkogu) [reply contracts/moneyflow/ether/WeiInfiniteFund.sol:11]: Because WeiInfiniteExpense will revert if amount is less then needed => revert always

Comment (AnthonyAkentiev) [reply contracts/moneyflow/ether/WeiInfiniteFund.sol:11]: ok, let's rename that to WeiUncappedFund.

And add comment "This fund will keep receiving ANY amount of ETH forever"

**Review by AnthonyAkentiev — APPROVED** [review]

**Pull request merged**